src/internal/common/slices.go

```diff
@@ -23,13 +21,18 @@ func First(vs ...string) string {
 	return ""
 }
 
-// parseBool returns the bool value represented by the string
-// or false on error
-func ParseBool(v string) bool {
-	s, err := strconv.ParseBool(v)
-	if err != nil {
+// Equal returns true if both slices contain the same
+// elements in the same order
+func AreSameSlice[T comparable](s1, s2 []T) bool {
```

Comment:
most/all of this function could be replaced by `reflect.DeepEqual`. According to the [doc](https://pkg.go.dev/reflect#DeepEqual), it will return false for `DeepEqual(nil, []bytes{})` or equivalent. However, that could be worked around with something like
```go
func AreSameSlice[T comparable](s1, s2 []T) bool {
  if len(s1) == len(s2) && len(s1) == 0 {
    return true
  }

  return reflect.DeepEqual(s1, s2)
}
```